### PR TITLE
8 password security backport

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

File: password_security/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+
+    'name': 'Password Security',
+    "summary": "Allow admin to set password security requirements.",
+    'version': '8.0.1.0.3',
```
Comment (lasley) [inline password_security/__openerp__.py]: 8.0.1.0.0

**Comment by bealdav** [comment]
ping @lasley

**Comment by lasley** [comment]
Hooray! I'll review today 😄

**Review by lasley — CHANGES_REQUESTED** [review]
Nice thanks @bealdav - one minor comment, plus akretion#9 to fix #674

**Review by lasley — APPROVED** [review]

**Review by moylop260 — APPROVED** [review]

**Comment by mourad-ehm** [comment]
:+1:

**Comment by sylvainc** [comment]
Hello,

Thanks, :+1: (code review)

**Pull request merged**